docs/importing_astropy.rst

```diff
@@ -1,16 +1,16 @@
-*********************************
-Importing astropy and subpackages
-*********************************
+**************************************
+Importing ``astropy`` and Sub-packages
+**************************************
 
-In order to encourage consistency amongst users in importing and using Astropy
+In order to encourage consistency among users in importing and using Astropy
 functionality, we have put together the following guidelines.
 
 Since most of the functionality in Astropy resides in sub-packages, importing
-astropy as::
+`astropy` as::
```

Comment:
```suggestion
``astropy`` as::
```

I'm not exactly sure why it triggers a warning from this file, but not from the other one.